game/requesthandler.cc

```diff
@@ -53,6 +64,7 @@ void RequestHandler::HandleFile(web::http::http_request request, std::string fil
 			content_type = "image/x-icon";
 		}
```

Comment:
Content type detection should be a function. Don't know if it is only uses here. Cause here is unchanged code, it is only a suggestion.